websites/management/commands/markdown_cleaning/cleaner.py

```diff
@@ -15,6 +15,26 @@
 from websites.utils import get_dict_field, set_dict_field
 
 
+def get_ocw_url(content: WebsiteContent):
+    """Return an ocw.mit.edu url to the given content."""
+    rootrel = get_rootrelative_url_from_content(content)
+    return f"https://ocw.mit.edu{rootrel}"
+
+
+def get_studio_url(content: WebsiteContent):
+    """Return an ocw-studio.odl.mit.edu url to the given content."""
+    site_name = content.website.name
+    if content.type == "sitemetadata":
+        return f"https://ocw-studio.odl.mit.edu/sites/{site_name}/type/metadata/"
+    return f"https://ocw-studio.odl.mit.edu/sites/{site_name}/type/page/edit/{content.text_id}/"
+
+
+def get_github_url(content: WebsiteContent):
+    """Return a github.mit.edu url to the given content."""
+    short_id = content.website.short_id
+    return f"https://github.mit.edu/mitocwcontent/{short_id}/tree/main/{content.dirpath}/{content.filename}.md"
+
```

Comment:
Might be better to use django settings for the domains instead of hardcoding them.  For example:
`OCW_STUDIO_BASE_URL` for the OCW Studio domain
`GIT_DOMAIN` for the github domain
`OCW_STUDIO_LIVE_URL` for ocwnext, though this isn't `ocw.mit.edu` on production (it's `https://ocw-published.odl.mit.edu/` instead - maybe that should be changed?) but should lead to the same place.